packages/discord.js/src/structures/Webhook.js

```diff
@@ -80,11 +81,15 @@ class Webhook {
     }
 
     if ('user' in data) {
-      /**
-       * The owner of the webhook
-       * @type {?(User|APIUser)}
-       */
-      this.owner = this.client.users?._add(data.user) ?? data.user;
+      if ('users' in this.client) {
+        this.owner = this.owner?._patch(data.user) ?? new User(this.client, data.user);
+      } else {
+        /**
+         * The owner of the webhook
+         * @type {?(User|APIUser)}
+         */
+        this.owner = this.owner ? { ...this.owner, ...data.user } : data.user;
```

Comment:
Couldn't we technically always create a user class? (we'd probably need to have a fake users manager on the webhook client I suppose..) Is that even worth it?

Reply:
Can't be a fake manager, but also can't be a real one without a some rework, i.e. for createDM, deleteDM